subsys/bluetooth/services/ots/ots_l2cap.c

```diff
@@ -72,7 +72,7 @@ static int ots_l2cap_send(struct bt_gatt_ots_l2cap *l2cap_ctx)
 	/* Mark that L2CAP TX was accepted. */
 	l2cap_ctx->tx.len_sent += len;
 
-	LOG_DBG("Sending TX chunk with %d bytes on L2CAP CoC", len);
+	LOG_DBG("Sending TX chunk with %d bytes on L2CAP CoC %d", len, l2cap_ctx->tx.len_sent);
```

Comment:
remove or add a description of what you are logging with `l2cap_ctx->tx.len_sent`

Reply:
Removed.